python/src/aiconfig/Config.py

```diff
@@ -53,13 +52,13 @@
 ModelParserRegistry.register_model_parser(
     DefaultAnyscaleEndpointParser("AnyscaleEndpoint")
 )
+ModelParserRegistry.register_model_parser(ClaudeBedrockModelParser())
+for model in gpt_models_extra:
```

Comment:
nit, would be better to have this colocate with the main registered ones above. Do you know why these are handled in a separate 'extra' array?

Reply:
this was a quick hack implemented in #994